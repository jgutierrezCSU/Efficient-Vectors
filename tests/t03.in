countOccurrences
20 11
9	4	9	9	1	2	6	3	7	6	4	4	5	9	2	9	7	8	3	1
countOccurrences
20 11
9	4	9	9	1	7	6	3	7	6	4	4	7	9	6	9	7	8	3	1
countOccurrences
50 11
10	7	0	0	1	5	7	7	3	7	1	6	3	1	8	6	8	0	5	3	6	3	5	5	7	8	3	5	10	9	1	4	5	9	4	9	7	0	8	3	10	8	8	8	7	10	0	6	3	0
