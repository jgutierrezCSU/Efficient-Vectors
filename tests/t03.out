9 4 9 9 1 2 6 3 7 6 4 4 5 9 2 9 7 8 3 1 
1 2
2 2
3 2
4 3
5 1
6 2
7 2
8 1
9 5
9 4 9 9 1 7 6 3 7 6 4 4 7 9 6 9 7 8 3 1 
1 2
3 2
4 3
6 3
7 4
8 1
9 5
10 7 0 0 1 5 7 7 3 7 1 6 3 1 8 6 8 0 5 3 6 3 5 5 7 8 3 5 10 9 1 4 5 9 4 9 7 0 8 3 10 8 8 8 7 10 0 6 3 0 
0 6
1 4
3 7
4 2
5 6
6 4
7 7
8 7
9 3
10 4
